### DISCUSSION TITLE
I want to get started in modifying shims. How do I do this?

### DISCUSSION THREAD

**Discussion by FunnyCorgi** [3kh0/ext-remover#1284] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I want to get into modifying shims. Not really to find exploits or bugs(though I might eventually start bug-hunting in shims), but just for fun and to learn about how it works.

**Comment by schoolexploitkid** [comment]
ask chatgpt for help xd, if its blocked then go cry

**Comment by FunnyCorgi** [comment]
No like seriously, I want to know from people who actually know what they're talking about(like coolelectronics, binbashbanana, and others who know what they are talking about), not chatgpt

**Reply by S-PScripts** [reply]
🔢

**Comment by awsome-webdev** [comment, marked as answer]
you do a fuckton of YouTube and googling that's how I learn things like that, you should look into google dorks too, they are very handy
